ogusa/tests/test_startyears.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+import os
+import ogusa
+from ogusa import SS, TPI
+
+CUR_PATH = os.path.abspath(os.path.dirname(__file__))
+PUF_PATH = os.path.join(CUR_PATH, '../puf.csv')
+
+def test_diff_start_year():
+    from ogusa.scripts.execute import runner
+    # Monkey patch enforcement flag since small data won't pass checks
+    SS.ENFORCE_SOLUTION_CHECKS = False
+    TPI.ENFORCE_SOLUTION_CHECKS = False
+    output_base = "./OUTPUT"
+    input_dir = "./OUTPUT"
+    for year in range(2016, 2020):
+        print('year = ', year)
+        user_params = {'frisch':0.41, 'debt_ratio_ss':1.0,
+                       'start_year': year}
+        runner(output_base=output_base, baseline_dir=input_dir, test=True,
+               time_path=True, baseline=True, age_specific=False,
+               user_params=user_params, run_micro=True,
+               small_open=False, budget_balance=False, data=PUF_PATH)
+
```

Comment:
@jdebacker Why did you choose years 2016 through 2020? Isn't the minimum year that can be specified 2013?

Also, it would be more efficient to just check the minimum and maximum years (something like 2013 and 2020) and just one or maybe two years in between. I don't think you would lose much if any test coverage if you were to do this. If, for example, 2013, 2017, and 2020 work, then I would expect 2014, 2015, 2016, 2018 and 2019 to work, too.

Reply:
Good call @hdoupe - will do this.  Thanks.